<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hero</string>
  </property>
  <widget class="QPushButton" name="btConnect">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>연결</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edCode">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>종목코드</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btOhlc">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OHLC</string>
   </property>
  </widget>
  <widget class="QTableView" name="tblOhlc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>731</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btConnect</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>connect_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>12</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btOhlc</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>ohlc_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>connect_clicked()</slot>
  <slot>ohlc_clicked()</slot>
 </slots>
</ui>
